src/openklant/components/klantinteracties/api/serializers/partijen.py

```diff
@@ -597,6 +616,75 @@ def update(self, instance, validated_data):
 
             validated_data["voorkeurs_digitaal_adres"] = voorkeurs_digitaal_adres
 
+        if "rekeningnummer_set" in validated_data:
```

Comment:
Ik dacht dat we hier alleen lezen van hadden gemaakt zodat dit soort constructies niet meer hoefde. Maar ik ga er nog eenmaal geheel door heen.

Reply:
Dit is voor de gemak van de gebruiker. Omdat we forceren dat de voorkeurs rekkeningnummer een gekoppelde rekkeningnummer is heb ik dit toegevoegd. Als we dit niet doen dan kan je niet in 1 keer een partij toevoegen met een voorkeurs rekkeningnummer.